README.md

```diff
@@ -1,13 +1,13 @@
-# cc4d
+# Produlytics
 Uno strumento per il controllo dei processi di produzione
 
 ## Struttura
-* **Candidatura**: contiene il documento con cui il gruppo si candida allo svolgimento del progetto didattico;
-* **RTB**: contiene i documenti per la revisione RTB;
-* **PB**: contiene i documenti per la revisione PB;
+* **Produlytics**: contiene il codice sorgente del prodotto;
 * **CA**: contiene i documenti per la revisione CA;
```

Comment:
Beh ma lo mettiamo anche se non c'è?

Reply:
Prima c'era. Anche secondo me si può tranquillamente togliere.

Reply:
se con prima intendi in rtb allora per me si può fare il merge. altrimenti lo toglierei

Reply:
Mi sa anche RTB, perché non credo che l'abbiamo mai cambiato